src/app/modules/group/pages/managed-groups/managed-groups.component.ts

```diff
@@ -7,6 +7,9 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ManagedGroupsComponent implements OnInit {
 
+  title = 'Groups you manage';
+  subtitle = 'Here are the groups you manage';
```

Comment:
🤨 If it is static, put it in the html directly

Reply:
I keept the title one, because I'm using it in other places now